homeassistant/components/starlink/translations/en.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
`string.json` is a source file for specific translations like this `en.json` file, yet, they seem to be out of sync.

Reply:
I see, have you got a link to any relevant documentation? I wasn't able to find much in the integration guide re. translations

Reply:
I have made an attempt to update strings, please let me know if I got anything wrong